std/assembly/typedarray.ts

```diff
@@ -1260,3 +1304,37 @@ function REVERSE<TArray extends ArrayBufferView, T>(array: TArray): TArray {
   }
   return array;
 }
+
+// @ts-ignore: decorator
+@inline
+function WRAP<TArray extends ArrayBufferView, T>(buffer: ArrayBuffer, byteOffset: i32 = 0, length: i32 = -1): TArray {
+  var bufferByteLength = buffer.byteLength;
+  if (byteOffset < 0 || byteOffset >= bufferByteLength) {
+    throw new RangeError(E_INDEXOUTOFRANGE);
+  }
+  var align = alignof<T>();
+  var byteLength: i32;
+  if (length < 0) {
+    if (length == -1) {
+      let size = 1 << align;
+      let mask = <i32>(size - 1);
```

Comment:
Making these `const` will lead to less locals due to precompute/inlining, I think.

Reply:
Fixed